### PR TITLE
feature (ref 35888): set targetBlank as an allow list

### PR DESCRIPTION
**Pull request by jalalifardemos**
**Ticket:** https://yaits.demos-deutschland.de/T35888

Description: Target Blank should set manually in HTMLPurifier_Config. The Key is HTML.TargetBlank,

### How to review/test
Code Review or test in UI

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

**Review by sakutademos — COMMENTED** [review]
It works now, thx :) But I cannot assess the backend part to determine if it's done correctly. So, we need approval from the backend

**Comment by jalalifardemos** [comment]
in [e2b21ab](https://github.com/demos-europe/demosplan-core/pull/2531/commits/e2b21ab677cbd346b9cfdebf6a5a15fd305b8ec1) `target` and `rel` used just for Links with those Attributes

File: demosplan/DemosPlanCoreBundle/Services/HTMLSanitizer.php [hunk 1]
```diff
@@ -89,6 +89,7 @@ public function purify(string $text): string
         $config = HTMLPurifier_Config::createDefault();
         $config->set('HTML.DefinitionID', 'dplan purifier');
         $config->set('HTML.DefinitionRev', 1);
+        $config->set('Attr.AllowedFrameTargets', ['_blank']);
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/Services/HTMLSanitizer.php:92]: To allow `rel` I think you have to add `Attr.AllowedRel` as well

Comment (jalalifardemos) [reply demosplan/DemosPlanCoreBundle/Services/HTMLSanitizer.php:92]: done in [058429b](https://github.com/demos-europe/demosplan-core/pull/2531/commits/058429b565a501c6ec554d63eacb56816f040e98)

**Review by mussbach — DISMISSED** [review]
lgtm, thx, code review only. You may need to change target branch

**Pull request merged**